lib/utilities/reportHandler.js

```diff
@@ -193,6 +193,12 @@ function getCRRStats(log, cb, _config) {
                 return done(null, retObj);
             }
         ),
+        retrieveRedisClients: done => {
+            if (process.env.REDIS_TEST === 'true') {
+                return backbeatMetrics.listClients(done);
```

Comment:
You want to return this only in the test mode because you are worried about revealing internals for security?

Reply:
It was more about not introducing a property that may cause the orbit api to reject the report due to bad schema